HTTP stress build is broken

Recent changes in MsQuic main probably added some dependencies/required parameters which are missing

https://dev.azure.com/dnceng-public/public/_build/results?buildId=657177&view=logs&j=22fea640-1099-5f32-ec5d-316ad83f359a&t=7fe64d4e-bb7a-5b68-0c5c-ad57d12d4da9&l=2333

```
CMake Error at CMakeLists.txt:67 (file):
  file STRINGS file "/etc/lsb-release" cannot be read.


CMake Error at CMakeLists.txt:68 (string):
  string sub-command REGEX, mode MATCH needs at least 5 arguments total to
  command.
```

--- a/src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile
+++ b/src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile
@@ -7,7 +7,7 @@ RUN apt-get update -y && \
     apt-get upgrade -y && \
     apt-get install -y cmake clang ruby-dev gem lttng-tools libssl-dev && \
     gem install fpm
-RUN git clone --recursive https://github.com/microsoft/msquic --depth 1
+RUN git clone --depth 1 --single-branch --branch v2.3.5 --recursive https://github.com/microsoft/msquic
 RUN cd msquic/ && \
     mkdir build && \
     cmake -B build -DCMAKE_BUILD_TYPE=Debug -DQUIC_ENABLE_LOGGING=false -DQUIC_USE_SYSTEM_LIBCRYPTO=true -DQUIC_BUILD_TOOLS=off -DQUIC_BUILD_TEST=off -DQUIC_BUILD_PERF=off -DQUIC_TLS=openssl3 -DQUIC_ENABLE_SANITIZERS=on && \

